### ISSUE TITLE
Simple

### ISSUE THREAD

**Issue by lhearachel** [lhearachel/plat-engine#31]
## Documentation

https://bulbapedia.bulbagarden.net/wiki/Simple_(Ability)

TL;DR: Simple now directly doubles the number of stages that a stat is increased or decreased by whenever a move, Ability, or item affects them, rather than treating each individual stage as 2 stages.

e.g., Defense Curl will directly give +2 and display "X's Defense sharply rose!", rather than giving +1, displaying "X's Defense rose!", and only acting as +2 during damage calculation.

## Relevant Code Points

Simple was already removed from the damage calculation process with the damage calc update.

Function `0x02242400` handles the code behind script command `StatBoostChange`; this function needs to be updated to check for Simple and boost the number of stages accordingly.
